src/ui/prompts/featurePrompts.ts

```diff
@@ -47,42 +46,37 @@ type SDKVisibilityChoice = {
     value: 'mobile' | 'client' | 'server',
     checked: boolean
 }
-type SDKVisibilityPromptResult = {
-    sdkVisibility: SDKVisibilityChoice['value'][]
-}
 
 export const getSDKVisibilityChoices = (sdkVisibility?: Feature['sdkVisibility']): SDKVisibilityChoice[] => {
     return [
         {
             name: 'mobile',
             value: 'mobile',
-            checked: sdkVisibility?.mobile || true
+            checked: sdkVisibility?.mobile !== undefined ? sdkVisibility?.mobile : true
```

Comment:
I think you should use `typeof 'undefined'` as the check instead in these cases

Reply:
good point, fixed!